templates/registry/markdown/entity_macros.j2

```diff
@@ -52,8 +52,9 @@
 | --- | --- | --- | --- | --- | --- | --- |{%- if id_attrs | length > 0 %}
 {{ attribute_table(id_attrs, "/docs/registry/attributes", e.lineage.attributes, notes, "identity") | trim }}{% endif %}{%- if desc_attrs | length > 0 %}
 {{ attribute_table(desc_attrs, "/docs/registry/attributes", e.lineage.attributes, notes, "description") | trim }}{% endif %}{%- if misc_attrs | length > 0 %}
-{{ attribute_table(misc_attrs, "/docs/registry/attributes", e.lineage.attributes, notes, "other") | trim }}
-{% endif %}
+{{ attribute_table(misc_attrs, "/docs/registry/attributes", e.lineage.attributes, notes, "other") | trim }}{% endif %}
```

Comment:
please revert this (it can be a follow-up PR)

you will get better responsiveness on these formatting PR reviews if you can make them easier to review